packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -189,8 +189,9 @@ Future<String?> getGradleVersion(
     // Expected content format (with lines above and below).
     // Version can have 2 or 3 numbers.
     // 'distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip'
+    // '^\s*' protects against commented out lines.
     final RegExp distributionUrlRegex =
-        RegExp(r'distributionUrl\s?=\s?.*\.zip');
+        RegExp(r'^\s*distributionUrl\s?=\s?.*\.zip', multiLine: true);
```

Comment:
Question (but nit even if yes), do we want to match even if there are multiple spaces around the equals? e.g. 
```
RegExp(r'^\s*distributionUrl\s*=\s*.*\.zip', multiLine: true)
```

Reply:
yes I will update this.